app/__init__.py

```
@@ -151,11 +142,13 @@ def nl2br(value):
     LoginForm,
     UserForm,
     EditUserForm,
-    DetalleRequisicionForm,
+    ConfirmarEliminarUsuarioForm,
+)
+from .requisiciones.forms import (
```

Flake8 (7.2.0)

[error] '.requisiciones.forms.RequisicionForm' imported but unused

(F401)